plugin/evm/vm.go

```diff
@@ -1082,6 +1083,16 @@ func (vm *VM) CreateHandlers(context.Context) (map[string]http.Handler, error) {
 		enabledAPIs = append(enabledAPIs, "subnet-evm-admin")
 	}
 
+	if vm.config.ValidatorsAPIEnabled {
+		validatorsAPI, err := newHandler("validators", &ValidatorsAPI{vm})
+		if err != nil {
+			return nil, fmt.Errorf("failed to register service for admin API due to %w", err)
+		}
+		apis[validatorsEndpoint] = validatorsAPI
+		enabledAPIs = append(enabledAPIs, "validators")
+	}
+
+	// RPC APIs
```

Comment:
nit: not sure which RPC this comment is referring to?

Reply:
`/rpc` endpoint